nowcasting_api/main.py

```diff
@@ -25,7 +24,7 @@
 
 structlog.configure(
     wrapper_class=structlog.make_filtering_bound_logger(
-        getattr(logging, os.getenv("LOGLEVEL", "INFO"))
+        getattr(logging, os.getenv("LOGLEVEL", "DEBUG"))
```

Comment:
do mind putting this back to INFO, just so when we deploy to production, its on INFO